tsrc/gitlab.py

```diff
@@ -98,13 +98,20 @@ def get_project_id(self, project_name):
                 raise
 
     def find_opened_merge_request(self, project_id, source_branch):
-        url = "/projects/%s/merge_requests" % project_id
+        url = "/projects/%s/merge_requests?state=opened" % project_id
         previous_mrs = self.make_request("GET", url)
         for mr in previous_mrs:
             if mr["source_branch"] == source_branch:
                 if mr["state"] == "opened":
                     return mr
 
+    def get_project_merge_request(self, project_id, mr_id):
+        mrs = self.make_request("GET", "/projects/%s/merge_requests" % (project_id))
+        for mr in mrs:
+            if str(mr['id']) == mr_id:
```

Comment:
I don't like this 'str()' cast

Reply:
What do you like?

you want it on both side?

how do I go from int to str?